flink/v1.15/flink/src/main/java/org/apache/iceberg/flink/source/IcebergSource.java

```diff
@@ -335,8 +365,30 @@ public Builder<T> exposeLocality(boolean newExposeLocality) {
       return this;
     }
 
+    /**
+     * Set the read properties for Flink source. View the supported properties in {@link
+     * FlinkReadOptions}
+     */
+    public Builder<T> set(String property, String value) {
+      readOptions.put(property, value);
+      return this;
+    }
+
+    /**
+     * Set the read properties for Flink source. View the supported properties in {@link
+     * FlinkReadOptions}
+     */
+    public Builder<T> setAll(Map<String, String> properties) {
```

Comment:
considering we already have `properties ` below, I am not sure if it is worth to add this one.

Reply:
To keep consistent with the sink style, `setAll` is retained and `properties` is marked as deprecated.